packages/boot/test/bootstrapTests/supports.js

```diff
@@ -495,3 +507,35 @@ export const makeSwingsetTestKit = async (
     timer,
   };
 };
+
+/**
+ * Start a SwingSet kernel to be used by Ava tests.  This kernel is expected to
+ * be shared across all tests in a given test module. By default Ava tests run
+ * in parallel, so be careful to avoid ordering dependencies between them.  For
+ * example, test accounts balances using separate wallets or test vault factory
+ * metrics using separate collateral managers. (Or use test.serial)
+ *
+ * The shutdown() function _must_ be called after the test is complete, or else
+ * V8 will see the xsnap workers still running, and will never exit (leading to
+ * a timeout error). Use t.after.always(shutdown), because the normal t.after()
+ * hooks are not run if a test fails.
+ *
+ * @param {import('ava').ExecutionContext} t
+ * @param {string} [bundleDir] directory to write bundles and config to
+ * @param {object} [options]
+ */
+export const makeSwingsetTestKit = async (t, bundleDir, options) =>
```

Comment:
I'm not sure the these wrappers are worth their weight.

Reply:
It annoyed me too, but what alternative is there?  It needed two different APIs, even though the underlying implementation is 99% common.

Reply:
alternative is to have consumers of these two APIs just use the base API and supply a logger

Reply:
Hmmm.  I was trying to stay backwards compatible, but given that there are only like 7 calls to `makeSwingsetTestKit`, changing every place it gets used actually doesn't sound that bad.

Reply:
Actually, that would help sever the dependency between `supports.js` and Ava, which might be good in and of itself.

Reply:
OK, done.